### ISSUE TITLE
Fix Keplr link to proposal

### ISSUE THREAD

**Issue by hakuno2000** [QuokkaStake/cosmos-proposals-checker#102]
I suggest you should change Keplr url format in `pkg/types/chain.go` like below

```
	if c.KeplrName != "" {
		links = append(links, Link{
			Name: "Keplr",
			Href: fmt.Sprintf("https://wallet.keplr.app/chains/%s/proposals/%s", c.KeplrName, proposalID),
		})
	}
```

For example, it should be
`https://wallet.keplr.app/chains/injective/proposals/418`
instead of
`https://wallet.keplr.app/#/injective/governance?detailId=418`

**Comment by freak12techno** [comment]
Actually yeah, that makes sense. Do you mind making a PR updating it to the correct one?
(I can also do it myself, but I have a bunch of things I should do before that, so it might take some time.

**Comment by freak12techno** [comment]
Okay I merged it and pushed v5.14.1 release which should contain it.
Also tested it manually, seems to work correctly.

@hakuno2000 thanks for your contribution!
I'll close this one as resolved, feel free to reopen if you think it's not working correctly, of file a new issue if there's something else.
